src/navigator.js

```diff
@@ -224,6 +224,8 @@ $.Navigator = function( options ){
     this.displayRegionContainer.appendChild(this.displayRegion);
     this.element.getElementsByTagName('div')[0].appendChild(this.displayRegionContainer);
 
+    this._navigatorRotate = options.navigatorRotate;
```

Comment:
We don't actually need to do this. All of the options are applied to the navigator on line 222.

Reply:
Ah yes, sorry. Didn't even see that. I resolved it in the newer commit.